packages/browser/src/integrations/dedupe.ts

```diff
@@ -1,5 +1,7 @@
-import { Event, EventProcessor, Exception, Hub, Integration, StackFrame } from '@sentry/types';
+import { Event, Exception, Integration, StackFrame } from '@sentry/types';
 import { logger } from '@sentry/utils';
+import { addGlobalEventProcessor } from '@sentry/hub';
+import { getHubAndIntegration } from '@sentry/integrations';
```

Comment:
Integrations is not a dep of browser, this needs to live in `@sentry/hub` probably